cmake/GraphicsSample.cmake

```diff
@@ -31,6 +31,17 @@ function(_add_sample_internal)
         add_executable("${TARGET_NAME}" ${ARG_SOURCES})
     endif()
 
+    if(PPX_BUILD_XR AND PPX_XR_QUEST)
+      # Quest devices do not use the standard OpenXR loader.
+      add_library(openxr_loader SHARED IMPORTED)
+      set_property(
+￼          TARGET
```

Comment:
When I tried to build the Quest variants, the spacing here caused issues for me. Re-formatting this locally fixed the issue for me. Is there a rogue whitespace character?

Reply:
Yes it is.. File is back to ascii-only! Thanks